src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Scanner;
 public class Duke {
+    public static Task[] list = new Task[100];
+    public static int numItems;
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public static final String LINE_BREAK = "\t____________________________________________________________";
+    public static final String YOUR_LIST = "\t_______________________~Your List~__________________________";
+    public enum TaskType{
+        TODO, DEADLINE, EVENT
+    }
+
+    public static void printList(){
+        System.out.println(YOUR_LIST);
+        for(int i=0; i<numItems; i++){
+            System.out.println("\t"+(i+1)+". "+ list[i]);
+        }
+        System.out.println(LINE_BREAK);
+    }
+    public static void addToList(String answer, TaskType taskType){
```

Comment:
Perhaps a more descriptive variable name for "answer" would be better here?

Reply:
I see, will incorporate the necessary changes! Thanks for your review :D